pkgs/tools/networking/connman/default.nix

```
@@ -99,17 +99,17 @@ stdenv.mkDerivation rec {
     # production build flags
     "--disable-maintainer-mode"
     "--enable-session-policy-local=builtin"
-    # This is for building and running tests (probably enabled by default),
-    # --enable-tests installs the tests as well
+    # for building and running tests
+    # --enable-tests # installs the tests, we don't want that
```

Sorry, that single `#` slipped in there, you can remove it again.
```suggestion
    # --enable-tests installs the tests, we don't want that
```

---

Ok, that's fine.